test-framework/oidc-server/src/main/java/io/quarkus/test/oidc/server/OidcWiremockTestResource.java

```diff
@@ -12,11 +12,7 @@
 import static java.util.stream.Collectors.joining;
 
 import java.security.cert.X509Certificate;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Could you configure your IDE to avoid star imports? We banned them from Quarkus (but couldn't find a way to configure our formatter to avoid them...).